### ISSUE TITLE
Entirely remove the cache system from discord-player-youtubei

### ISSUE THREAD

**Issue by retrouser955** [retrouser955/discord-player-youtubei#10]
# TLDR;

The current cache system provided by youtubei.js is unreliable and causes a lot of issues.

# Why remove the cache?

youtubei.js seems to like to break due to cache issues. In version `1.0.9`, we temporarily had to use the ANDROID platform to get streams. This is due to a problem with the cache. In version `1.1.4`, playlist URLs stopped working. Again, due to cache issues. On top of this, discord-player already provide a non-persistent cache for use which makes the cache implementation of youtubei.js's cache entirely useless. 

# Conclusion

Due to having faced issues, I am planning to remove the cache in this module that is from youtubei.js.

**Comment by febkosq8** [comment]
Given that this package is supposed to work with discord-player anyway. 
Agree that it makes sense to use cache system that this package implements.

**Comment by retrouser955** [comment]
Cache system is disabled as of `1.1.5` and will be entirely removed by `1.2.x`
